# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glu/hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glu-9.0.0/src/libutil/error.c.vanilla	2020-02-01 14:50:28.808090639 +0100
+++ glu-9.0.0/src/libutil/error.c	2020-02-01 14:50:45.460090836 +0100
@@ -28,6 +28,7 @@
  * Silicon Graphics, Inc.
  */
 
+#include <stddef.h>
 #include "gluos.h"
 #include "gluint.h"
 #include <GL/glu.h>
